src/classes/VOL_CTRL_VolunteersBulkEnterHours_TEST.cls

```diff
@@ -121,11 +121,11 @@ private with sharing class VOL_CTRL_VolunteersBulkEnterHours_TEST {
         // test the save!
         ctrl.SaveVolunteerHours();
         system.debug(ctrl.strSaveResults);
-        System.Assert(ctrl.strSaveResults.startsWith('Error'));
+        System.assertNotEquals(null, ctrl.strSaveResults);
```

Comment:
@sam-knox - This assert was previously checking that an error was displayed not just that the save result is not null. I'm wondering if we change it to state that it contains the label we expect to see here instead?

Reply:
@jjbennett Sam made this change on my advice. Looking at the controller, strSaveResults is only set to a non-null value if there was an error (unless I missed something). We thought it was better to test for the boolean condition of whether there was an error rather than the specific error text (since a change in the error text had broken the test).